webapp/channels/src/components/custom_status/custom_status_emoji.tsx

```diff
@@ -66,44 +64,46 @@ function CustomStatusEmoji({
     }
 
     return (
-        <OverlayTrigger
-            delayShow={Constants.OVERLAY_TIME_DELAY}
-            placement={tooltipDirection}
-            overlay={
-                <Tooltip id='custom-status-tooltip'>
+        <WithTooltip
+            id='postEmoji__tooltip'
+            title={
+                <>
                     <div className='custom-status'>
-                        <RenderEmoji
+                        {/* <RenderEmoji
                             emojiName={customStatus.emoji}
                             size={14}
                             emojiStyle={{
                                 marginTop: 2,
                             }}
-                        />
-                        {customStatus.text &&
+                        /> */}
```

Comment:
We can remove this if no longer needed to avoid dead code. 

Also, can you check the spacing below? The linter seems to be tripping on indentation